samples/spring-boot/AppConfigExternal.pkl

```diff
@@ -0,0 +1,15 @@
+// this module name determines the package and
+// class name of the generated Java config class
+module samples.boot.AppConfigExternal
+
+server2: Server
+
+class Server {
+  endpoints: Listing<Endpoint>
+}
+
+class Endpoint {
+  name: String
+  port: UInt16
+  externalConfig: Boolean = true
+}
```

Comment:
I think adding this to the sample can end up misleading users; in a typical application, your application schema would just be part of your modulepath.

I think let's either just remove this, or create a new sample for this (you can copy/paste most of the existing sample).